Telerik.Examples.Mvc/Telerik.Examples.Mvc/Views/Grid/Grid_SignalR.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿<script src="https://unpkg.com/@@aspnet/signalr@1.1.0/dist/browser/signalr.js"></script>
+<script>
+    var url = "/gridHub";
+
+    var hub = new signalR.HubConnectionBuilder()
+        .withUrl(url, {
+            transport: signalR.HttpTransportType.LongPolling
+        })
+        .build();
+
+    var hubStart = hub.start();
+
+
+    function onEdit() {
+        this.showColumn(2);
+    }
+
+    function onSave() {
+        this.hideColumn(2);
+    }
+</script>
+
+<div class="signalr-grid-container">
+    @(Html.Kendo().Grid<Telerik.Examples.Mvc.Models.ProductViewModel>()
+        .Name("grid")
+        .Columns(columns =>
+        {
+            columns.Bound(p => p.ProductName).Width(150);
+            columns.Bound(p => p.UnitsInStock).Width(120).Filterable(false);
+            columns.Command(command =>
+            {
+                command.Edit();
+                command.Destroy();
+            }).Width(80);
+        })
+        .ToolBar(toolbar =>
+        {
+            toolbar.Create();
+        })
+        .Groupable()
+        .Pageable()
+        .Editable(editable => editable.Mode(GridEditMode.InLine))
+        .Scrollable()
+        .HtmlAttributes(new { style = "height:100%;" })
+        .DataSource(dataSource => dataSource
+            .SignalR()
+            .PageSize(20)
+            .Transport(tr => tr
+                .Promise("hubStart")
+                .Hub("hub")
+                .Client(c => c
+                    .Read("read")
+                    .Create("create")
+                    .Update("update")
+                    .Destroy("destroy")
+                )
+                .Server(s => s
+                    .Read("read")
+                    .Create("create")
+                    .Update("update")
+                    .Destroy("destroy")
+                )
+            )
+            .Schema(schema => schema
+                .Model(model =>
+                {
+                    model.Id("ID");
```

Comment:
There is no `ID` field marked as a unique identifier. I would recommend updating the schema with [ProductID](https://github.com/telerik/ui-for-aspnet-core-examples/pull/82/files#diff-959076d6a24fe75ce2fd528ae45f5f92289d0afb2904c4668c91de0af360194bR11) instead.